### PR TITLE
CC-15100 Release fixes

### PR DESCRIPTION
**Pull request by kkicheglovspryker**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,39 +6,42 @@
     "require": {
         "php": ">=7.4",
         "spryker-eco/unzer-api": "dev-feature/cc-14610/dev-unzer-psp as 1.0",
-        "spryker-shop/shop-ui": "^1.56.0",
         "spryker/calculation": "^4.11.0",
         "spryker/cart-extension": "^4.1.0",
         "spryker/checkout-extension": "^1.3.0",
-        "spryker/customer": "^7.0.0",
-        "spryker/kernel": "^3.30.0",
```
Comment (AsonUnique) [inline composer.json:14]: Are you sure you don't need `spryker/kernel` in your dependencies?

File: phpstan.neon [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+    paths:
+        - src/
     bootstrapFiles:
         - tests/bootstrap.php
-
-    treatPhpDocTypesAsCertain: false
+    checkGenericClassInNonGenericObjectType: false
+    excludePaths:
     ignoreErrors:
+        - '#Call to method .+\(\) on an unknown class Generated\\.+\\Ide\\AutoCompletion.#'
```
Comment (AsonUnique) [inline phpstan.neon]: This was fixed in https://github.com/spryker/kernel/releases/tag/3.61.1

**Review by michbeck — APPROVED** [review]

**Review by AsonUnique — APPROVED** [review]

File: src/SprykerEco/Zed/Unzer/Business/Payment/Processor/PreparePayment/UnzerPreparePaymentProcessor.php [hunk 3]
```diff
@@ -59,14 +59,11 @@ public function prepareUnzerPaymentTransfer(QuoteTransfer $quoteTransfer, SaveOr
...
-
         $unzerBasket = $this->createUnzerBasket($quoteTransfer, $unzerPaymentTransfer->getUnzerKeypairOrFail());
-        $amountTotal = (float)$quoteTransfer->getTotalsOrFail()->getGrandTotal() / UnzerConstants::INT_TO_FLOAT_DIVIDER;
-        $unzerPaymentTransfer->setBasket($unzerBasket);
-        $unzerPaymentTransfer->setCurrency($quoteTransfer->getCurrencyOrFail()->getCode());
-        $unzerPaymentTransfer->setAmountTotal($amountTotal);
 
-        return $unzerPaymentTransfer;
+        return $unzerPaymentTransfer->setBasket($unzerBasket)
+            ->setCurrency($quoteTransfer->getCurrencyOrFail()->getCode())
```
Comment (DmitryLymarenko) [inline src/SprykerEco/Zed/Unzer/Business/Payment/Processor/PreparePayment/UnzerPreparePaymentProcessor.php:65]: please update API specs with required transfer properties

**Review by DmitryLymarenko — APPROVED** [review]

**Comment by dereuromark** [comment]
Please always remove branches after merge etc!

**Pull request merged**